troubleshooting/faq.md

```diff
@@ -161,6 +162,41 @@ So it’s important to understand the different storage driver options available
 
 We recommend that you use the `overlay` driver, rather than `devicemapper`; for help configuring this [please see the following tutorial](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/#configure-docker-with-the-overlayoverlay2-storage-driver)
 
+## What should I do if ssl problem occurs with npme over https?
+
+While working with custom certificate for npme registry, you might get an error related to ssl certificate on https while logging in or publishing a package, but it works fine on browser with https. It happens because node is a bit stricter about certificates than Chrome.
```

Comment:
Minor changes: 
 > When using a custom certificate for your npme registry, you might get an error related to the SSL certificate when logging in or publishing a package even though the browser does not display errors when viewing the npme registry site. This is because node's certificate validation is strict and doesn't allow for things like self-signed certificates by default.